src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -58,4 +73,23 @@ public void startExecuting()
 
         citizen.getNavigator().tryMoveToXYZ((double) pos.getX() + 0.5D, (double) pos.getY(), (double) pos.getZ() + 0.5D, 1.0D);
     }
+
+    /**
+     * While going home play a goHome sound for the specific worker by chance.
+     */
+    private void playGoHomeSounds()
+    {
+        Random rand = new Random();
+
+        int chance = rand.nextInt(ONE_IN_A_THOUSAND);
+
+        if(chance <= 1)
+        {
+            if (citizen.getWorkBuilding() != null && ("fisherman").equals(citizen.getWorkBuilding().getJobName()))
+            {
+                SoundUtils.playSoundAtCitizenWithChance(citizen.worldObj, citizen.getPosition(), FishermanSounds.Female.offToBed, 1);
+            }
+            //todo add for other workers as soon as available.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)